RecoTracker/IterativeTracking/python/LowPtTripletStep_cff.py

```diff
@@ -294,6 +294,7 @@
         mva = dict(GBRForestLabel = 'HIMVASelectorLowPtTripletStep_Phase1'),
         qualityCuts = [-0.8, -0.4, 0.5],
 )
+
```

Comment:
remove errant newline so this PR doesn't change this file for no reason

Reply:
The offending newline is removed in the latest commit